tools/memap.py

```diff
@@ -18,7 +23,7 @@
 from jinja2 import FileSystemLoader, StrictUndefined
 from jinja2.environment import Environment
 
-from .utils import (argparse_filestring_type, argparse_lowercase_hyphen_type,
+from tools.utils import (argparse_filestring_type, argparse_lowercase_hyphen_type,
                     argparse_uppercase_type)
```

Comment:
Correct indentation please.

Reply:
Ok, I updated it to match `build_api.py`.